rtc fails to build for msb-430

Missing header:

```
RIOT/drivers/include/rtc.h:27:22: fatal error: sys/time.h: No such file or directory`
```

Without that import:

```
RIOT/drivers/include/rtc.h:60:1: error: unknown type name 'time_t'
RIOT/drivers/include/rtc.h:60:24: warning: 'struct timeval' declared inside parameter list [enabled by default]
RIOT/drivers/include/rtc.h:60:24: warning: its scope is only this definition or declaration, which is probably not what you want [enabled by default]
```

--- a/sys/auto_init/auto_init.c
+++ b/sys/auto_init/auto_init.c
@@ -16,14 +16,27 @@
  */
 #include <stdint.h>
 #include <stdio.h>
+
+#include "auto_init.h"
+
+#ifdef MODULE_UART0
 #include "board_uart0.h"
-#include "rtc.h"
+#endif
+
+#ifdef MODULE_MCI
 #include "diskio.h"
-#include <auto_init.h>
+#endif
+
+#ifdef MODULE_VTIMER
 #include "vtimer.h"
+#endif
+
+#ifdef MODULE_RTC
+#include "rtc.h"
+#endif
 
 #define ENABLE_DEBUG (0)
-#include <debug.h>
+#include "debug.h"
 
 extern int main(void);
 

